Source/ACE.Server/WorldObjects/Player.cs

```diff
@@ -47,6 +47,11 @@ public partial class Player : Creature, IPlayer
 
         public float CurrentRadarRange => Location.Indoors ? 25.0f : 75.0f;
 
+        public TimeSpan PlayerSaveInterval
```

Comment:
This can go back in Player_Database.cs

Reply:
Woops. This isn't supposed to be in here anymore. Thanks for catching that.

Reply:
I remember what happened here. It says `PlayerManager` is not in scope for Player_Database